### ISSUE TITLE
workflow for editing page

### ISSUE THREAD

**Issue by ronkeizer** [isop-phmx/awesome-pharmacometrics#2]
Just checking whether I should I edit the HTML or the params.json file?
If the HTML is compiled from the JSON, what tool are you using for that? (And in that case, why are all carriage returns absent?)
cheers

**Comment by vjd** [comment]
@ronkeizer -  @dpastoor and I discussed about the best workflow for this when we created this page, but realized that it may be just easier to work off the main github page as a `Readme.md`and then transfer that to a html file. 
We don't have a solution currently compile the json to html .. do you have any ideas?

**Comment by ronkeizer** [comment]
Yes, was wondering what the params.json was, perhaps it's created by Github then.
I propose this: in the pull request I'll submit in a minute I've included a few new files. Now you edit the page by editing the contents.md file. Once you're done you run 

```
bash compile.sh
```

which uses pandoc to convert the markdown into HTML and adds the header and footer. Then you push to GitHub pages and the new page is on-line. This way we don't have to bother with writing any HTML. OK?

**Comment by smouksassi** [comment]
http://stackoverflow.com/questions/16386195/github-pages-wheres-the-markdown 

seems we are not the only one trying to figure this out.

**Comment by vjd** [comment]
did the pull request come in? Maybe @dpastoor  or @smouksassi  can approve it

**Comment by ronkeizer** [comment]
OK. We can use jekyll too, like suggest in the SO post. I'm using that for my blog and works fine, but not sure if you all have Ruby and jekyll installed. What I suggested above might be quicker.

**Comment by smouksassi** [comment]
I dont know use these tools wow I have tons of thing to learn to catch up with web technologies ! :)

**Comment by dpastoor** [comment]
Yeah @ronkeizer I was thinking along the lines of jekyll too. 

I think the conundrum is figuring out a way to have pull requests come only through the README.md like the other awesome-x repos, making it very easy for people to contribute, but also give a github-pages style actual website for people that are just want to see a website via a URL rather than going to a github readme. 

A shell script etc is not ideal as it still has to be executed locally to initiate the change, rather than just accepting a pull request and being done with it. I was thinking along the lines of a hook that re-ran every time a new commit was accepted - a la travisCI

**Comment by ronkeizer** [comment]
sure that would be nice, please go ahead and set that up. However, at this point I think it's most important to just get started, rather than getting this thing fully automated. So if you don't get this running now or don't have time for it, I think it's best to just start with the shell script so we can start filling in stuff. We can always drop in the Jekyll + Github hooks later.

**Comment by dpastoor** [comment]
I agree - and i don't think the best method is via actually dealing with html - as such - to make everyone's lives easier lets just work directly with the README.md on the master branch - very easy to issue pull requests, even directly from github

**Comment by dpastoor** [comment]
@ronkeizer the readme should be stable enough (regarding formatting) to start adding content - I looked more into the jekyll documentation, and it should be fine. My biggest 'concern' was that it needed a specific folder structure, like how we have our blogs set up, which makes it harder for a random person to issue a pull request. 

From the docs looks like there is enough it can work with to inject the content from the readme into the body of the website template without having to do anything locally
